.github/workflows/ruby.yml

```
@@ -7,13 +7,15 @@ jobs:
       matrix:
         gemfile:
           - Gemfile
+          - gemfiles/Gemfile-rails-7-1
```

It seems this should be:
```suggestion
          - gemfiles/Gemfile-rails-7-2
          - gemfiles/Gemfile-rails-7-1
```